lldb/test/API/functionalities/data-formatter/data-formatter-stl/generic/coroutine_handle/TestCoroutineHandle.py

```diff
@@ -78,8 +78,19 @@ def do_test(self, stdlib_type):
                 ],
             )
 
-        # Run until after the `co_yield`
         process = self.process()
+
+        # Break at a coroutine body
+        lldbutil.continue_to_source_breakpoint(
+          self, process, "// Break at co_yield", lldb.SBFileSpec("main.cpp", False)
```

Comment:
I'd suggest breaking on `int_generator my_generator_func`. Or something that's not a comment, since they aren't a reliable way to generate valid break locations

Reply:
Do you mean we run the lldb command like `self.runCmd("break my_generator_func")` to stop at a break point?

BTW, this file is using this `continue_to_source_breakpoint ` pattern in other places, I was following this pattern.

Reply:
I meant you can just do
```
 lldbutil.continue_to_source_breakpoint(
          self, process, "int_generator my_generator_func", lldb.SBFileSpec("main.cpp", False)
```
Or sometimes we put something like `std::puts("Break here");` into the source, instead of relying on comments.

> BTW, this file is using this continue_to_source_breakpoint  pattern in other places, I was following this pattern.

Yup, it isn't very strictly enforced. Also it isn't part of the LLDB testing guidelines. Maybe it should be

Reply:
I see, thanks for the explanation. Fixed now.